### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by AIn0n**
added instructions to run app in readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Web application - exchange rate app
+
+App prepared for ZPI course at TUL.
+
+## Team
+
+| position | name | github nick |
```
Comment (Panzer0) [inline README.md:7]: "github nickname" would be more appropriate

File: README.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+pip install -r requirements.txt
+```
+after that you can run application with command:
+```bash
+make run
+```
+
+## Dev ops platform
+
+We are using github actions as a main place for developing pipeline - mostly for running automated tests before each merge. This give us sanity and insure as that every new commit will not break any existing functionality. You can check the details [here](https://github.com/IIS-ZPI/ZPI2021_IO1_Banana_eaters/actions).
```
Comment (Panzer0) [inline README.md:31]: I would rephrase this as 

> This gives us the peace of mind and ensures that no existing functionality is broken by new commits.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+## MVP
+Functionalities demanded for minimal value product are list in this [file]().
```
Comment (Panzer0) [inline README.md]: > Functionalities demanded for the minimal value product are listed in this

**Review by Panzer0 — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (AIn0n) [reply README.md]: check it now

File: README.md (hunk 2 above)
Comment (AIn0n) [reply README.md:35]: Ok

File: README.md (hunk 3 above)
Comment (Panzer0) [reply README.md]: Seems fine!
